driver/ixsystems/common.py

```diff
@@ -266,35 +263,33 @@ def _create_iscsitarget(self, name, volume_name):
         self._target_to_extent(tgt_id, ext_id)
 
     def delete_target_to_extent(self, tgt_ext_id):
-        pass
+        """Deletes specified iSCSI target extent."""        
```

Comment:
Remove unnecessary pass

Reply:
This should raise a NotImplementedError if this is called without an implementation, no?

Reply:
Thank you for point out. I double checked, I found out NotImplementedError will be thrown for none implemented, abstract method. This method is empty there might be caused by freenas api 1.0 to 2.0 change legacy, and it is not abstract class method.

Reply:
I double checked again, it is quite confusing indeed.  But it works without pass and it is forced by pylint pep8 prompt 
Here is the test code I created, it works fine:
class ca :
    
    def t(self):
        '''doc'''
    def b(self):
        '''doc'''
if __name__ == "__main__":
    a = ca()
    test_call = a.t()
    print("ok")
Output: ok

Reply:
I believe it is legacy code for freenas api 1.0, at that time, need those method delete extend before delete zvol, on truenas api 2.0, delete zvol auto delete iscsi target and extend hence leave it empty there....

Reply:
Do we still support 1.0 api in this code?

Reply:
The 1.0 api support has been removed since last year. Freenas >12.0 and above.

Reply:
Ok, so this method should be removed altogether?